zen-quotes-app/src/app/home/home.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
+
+import { Quote } from '../../types/quote.type';
+import { QuotesService } from '../quotes.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  title = 'Inspiration';
+  quotes: Quote[] = [];
+  authors: string[] = [];
+  authorsFilter: string[] = [];
+  randomQuote: Quote = { author: '', text: '' };
+  quotesFiltered: Quote[] = [];
+  quotesPerPage = 9;
+  pagesArray: number[] = [];
+  activePage = 0;
+
+  constructor(private quotesService: QuotesService) {}
+
+  getAuthors(quotes: Quote[]) {
+    return [...new Set(quotes.map((quote) => quote.author))].sort();
+  }
+
+  setAuthors(authors: string[]) {
+    this.authors = authors;
+  }
+
```

Comment:
Hi @jvandemo 🙌

Can it be this way about authors' function? 
I am sorry if I am wrong again 🙏

Thank you in advance! 🙇‍♀️

Reply:
@IraLysenko Yes, the `setAuthors` function looks great, nice work! 👏 👏

Reply:
Horray! Thank you very much! 🙌🙇‍♀️